### DISCUSSION TITLE
Why use `r`, `s`, `c` as thing names instead of `(`/`)`, `[`/`[`, `{`/`}`?

### DISCUSSION THREAD

**Discussion by leira** [meow-edit/meow#295] [Q&A] [answered]
I found it a bit strange to not use `(`/`)`, `[`/`[`, `{`/`}` as thing names directly, but to use `r`, `s`, `c` instead? Is it a limitation on Emacs, or a design decision?

**Comment by DogLooksGood** [comment, marked as answer]
round, square and curly. It's customisable.

**Reply by leira** [reply]
Do you mean it is because of style consistency rather than limitations? I can remap them to `(`/`)`, `[`/`]`, `{`/`}`? Let me give it a try. Thanks!
